docs/get_started/getting_started_riscv_cmake.md

```diff
@@ -17,15 +17,18 @@ executables that can be run on the target platform.
 
 ### Install RISC-V Tools
 
-Execute the following script to download RISC-V toolchain and QEMU:
+Execute the following script to download the prebuilt RISC-V toolchain and QEMU:
 
 ```shell
 # In IREE source root
 $ ./build_tools/riscv/riscv_bootstrap.sh
 ```
 
-* RISC-V toolchain is built from https://github.com/llvm/llvm-project
-* RISC-V QEMU is built from https://github.com/sifive/qemu/tree/v5.1.0-rvv-zfh-pmp
+NOTE: If you want to build the tools by yourself:
+* RISC-V toolchain is built from https://github.com/llvm/llvm-project (main branch).<br>
+-- Currently, our LLVM compiler is built on GNU toolchain, including libgcc, GNU linker, and C libraries. You need to build GNU toolchain first.<br>
+-- Clone GNU toolchain from: https://github.com/riscv/riscv-gnu-toolchain (master branch). Switch the "riscv-binutils-gdb" submodule to `rvv-1.0.x-zfh ` branch.
```

Comment:
I guess this is for `riscv-binutils` submodule. https://github.com/riscv/riscv-binutils-gdb is used in two submodules in `.gitmodules`

Reply:
Yes. Let me modify it as "riscv-binutils" submodule. No need to change "riscv-gdb"

Reply:
Thanks for the update.